### PR TITLE
exmo - fetchTransactionFees: codes update

### PR DESCRIPTION
**Pull request by AlgoQ**

### CODE REVIEW

File: js/exmo.js [hunk 1]
```diff
@@ -472,25 +474,29 @@ module.exports = class exmo extends Exchange {
         //         ],
         //     }
         //
-        const result = {
-            'info': cryptoList,
-            'withdraw': {},
-            'deposit': {},
-        };
+        const result = {};
```
Comment (carlosmiei) [inline js/exmo.js:477]: @AlgoQ Do we need this extra call?
image

Can you try to replace this with `this.currencies` ?

**Comment by frosty00** [comment]
i think we should just iterate over the currencies here instead of repeating ourselves with the same parsing logic twice

**Comment by AlgoQ** [comment]
You mean just iterating over the `cryptoList`, right?

**Comment by carlosmiei** [comment]
LGTM ✅ 

```
p exmo fetchTransactionFees '["USDT"]'
Python v3.9.7
CCXT v2.0.62
exmo.fetchTransactionFees(['USDT'])
{'USDT': {'deposit': 0.0,
          'info': [{'comment': 'Minimum deposit amount is 10 USDT',
                    'commission_desc': '0%',
                    'currency_confirmations': '10',
                    'currency_name': 'USDT',
                    'enabled': False,
                    'max': '0',
                    'min': '10',
                    'name': 'USDT (OMNI)',
                    'type': 'deposit'},
                   {'comment': 'Do not withdraw directly to the Crowdfunding '
                               'or ICO address as your account will not be '
                               'credited with tokens from such sales.',
                    'commission_desc': '5 USDT',
                    'currency_confirmations': '10',
                    'currency_name': 'USDT',
                    'enabled': False,
                    'max': '100000',
                    'min': '10',
                    'name': 'USDT (OMNI)',
                    'type': 'withdraw'},
                   {'comment': 'Minimum deposit amount is 10 USDT. We do not '
                               'support BSC, Poligon and BEP20 network, please '
                               'consider this when sending funds',
                    'commission_desc': '0%',
                    'currency_confirmations': '10',
                    'currency_name': 'USDT',
                    'enabled': True,
                    'max': '0',
                    'min': '10',
                    'name': 'USDT (ERC20)',
                    'type': 'deposit'},
                   {'comment': 'Caution! Do not withdraw directly to a '
                               'crowdfund or ICO address, as your account will '
                               'not be credited with tokens from such sales.',
                    'commission_desc': '5 USDT',
                    'currency_confirmations': '10',
                    'currency_name': 'USDT',
                    'enabled': True,
                    'max': '200000',
                    'min': '100',
                    'name': 'USDT (ERC20)',
                    'type': 'withdraw'},
                   {'comment': 'Minimum deposit amount is 1 USDT. Only TRON '
                               'main network supported',
                    'commission_desc': '0%',
                    'currency_confirmations': '10',
                    'currency_name': 'USDT',
                    'enabled': True,
                    'max': '100000',
                    'min': '1',
                    'name': 'USDT (TRC20)',
                    'type': 'deposit'},
                   {'comment': 'Caution! Do not withdraw directly to a '
                               'crowdfund or ICO address, as your account will '
                               'not be credited with tokens from such sales. '
                               'Only TRON main network supported.',
                    'commission_desc': '1 USDT',
                    'currency_confirmations': '10',
                    'currency_name': 'USDT',
                    'enabled': True,
                    'max': '150000',
                    'min': '10',
                    'name': 'USDT (TRC20)',
                    'type': 'withdraw'}],
          'withdraw': 1.0}}
```

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**